packages/google_maps_flutter/google_maps_flutter_ios/ios/google_maps_flutter_ios.podspec

```diff
@@ -22,8 +22,15 @@ Downloaded by pub (not CocoaPods).
   # has been confirmed to be compatible via an example in examples/. See discussion
   # in https://github.com/flutter/flutter/issues/86820 for why this is so broad.
   s.dependency 'GoogleMaps', '< 9.0'
+  # Once the plugin requires iOS version 13+, this should be updated.
```

Comment:
This day has come :) We now require iOS 14+.

Reply:
Note that the clustering still works with the version 4.1.0 as well, so updating it might not be mandatory for now.

Reply:
Sounds like this is blocked on https://github.com/googlemaps/google-maps-ios-utils/issues/456 then.

I would image that will be fixed soon, because Google Maps 7.3 isn't compliant with the upcoming App Store privacy requirements, so the current version of Google-Maps-iOS-Utils will become essentially unshippable once enforcement of those requirements starts.

Reply:
@stuartmorgan FYI that repository you linked does not seem to be actively maintained

Reply:
@cedvdb See https://github.com/googlemaps/google-maps-ios-utils/issues/473